packages/ui/src/views/delegates/components/delegate-connectivity-list.tsx

```diff
@@ -0,0 +1,102 @@
+import { useEffect } from 'react'
+
+import { Badge, Icon, NoData, SkeletonList, SkeletonTable, Table } from '@/components'
+import { cn } from '@utils/cn'
+import { timeAgo } from '@utils/utils'
+import { defaultTo } from 'lodash-es'
+
+import { DelegateConnectivityListProps } from '../types'
+
+const Title = ({ title }: { title: string }): JSX.Element => (
+  <span className="max-w-full truncate font-medium">{title}</span>
+)
+
+export function DelegateConnectivityList({
+  delegates,
+  useTranslationStore,
+  isLoading,
+  selectedTags,
+  isDelegateSelected
+}: DelegateConnectivityListProps): JSX.Element {
+  const { t } = useTranslationStore()
+
+  if (isLoading) {
+    return <SkeletonList />
+  }
+
+  if (!delegates.length) {
+    return (
+      <NoData
+        withBorder
+        iconName="no-data-cog"
+        title={t('views:noData.noDelegates', 'No delegates yet')}
+        description={[t('views:noData.noDelegates', 'There are no delegates in this project yet.')]}
+      />
+    )
+  }
+
+  return (
+    <Table.Root
+      className={isLoading ? '[mask-image:linear-gradient(to_bottom,black_30%,transparent_100%)]' : ''}
+      variant="asStackedList"
+    >
+      <Table.Header>
+        <Table.Row>
```

Comment:
move strings to views.json

Reply:
will do a follow up for this, is required in multiple places